worker/worker.go

```diff
@@ -1,4 +1,6 @@
-// Copyright (c) 2017 Uber Technologies, Inc.
+// Copyright (c) 2017-2020 Uber Technologies, Inc.
```

Comment:
Is this the new format we should use for all new files? What about the server repo?

Reply:
Updated with the latest license information from legal.

Reply:
Only if we are merging changes from the temporal repo, and for files that got changed. We haven't ported anything in server yet so no need to change license.